src/Chainweb/Pact/Backend/ChainwebPactDb.hs

```diff
@@ -207,7 +207,7 @@ recordPendingUpdate (Utf8 key) (Utf8 tn) txid v = modifyPendingData modf
     deltaKey = SQLiteDeltaKey tn key
 
     modf (a, m, l, p) = (a, upd m, l, p)
-    upd = HashMap.insertWith DL.append deltaKey (DL.singleton delta)
+    upd = HashMap.insertWith const deltaKey (DL.singleton delta)
```

Comment:
Just add a comment here that the "const" means we prefer the latest update for the roekey for the current transaction.

Reply:
done